keras_hub/src/models/retinanet/prediction_head.py

```diff
@@ -0,0 +1,148 @@
+import keras
+
+
+class PredictionHead(keras.layers.Layer):
+    """The classification/box predictions head.
+
+    Arguments:
+        output_filters: int. Number of convolution filters in the final layer.
+        num_filters: int. Number of convolution filters used in base layers.
+            Defaults to `256`.
+        num_conv_layers: int. Number of convolution layers before final layer.
+            Defaults to `4`.
+        kernel_initializer: `str` or `keras.initializers` initializer.
+            The kernel initializer for the convolution layers.
+            Defaults to `"random_normal"`.
+        bias_initializer: `str` or `keras.initializers` initializer.
+            The bias initializer for the convolution layers.
+            Defaults to `"zeros"`.
+        kernel_regularizer: `str` or `keras.regularizers` regularizer.
+            The kernel regularizer for the convolution layers.
+            Defaults to `None`.
+        bias_regularizer: `str` or `keras.regularizers` regularizer.
+            The bias regularizer for the convolution layers.
+            Defaults to `None`.
+
+    Returns:
+      A function representing either the classification
+        or the box regression head depending on `output_filters`.
+    """
+
+    def __init__(
+        self,
```

Comment:
args defining the architecture configuration should not have the default value set.

Reply:
+1

Reply:
Done